src/components/Header.js

```diff
@@ -1,12 +1,12 @@
 import "./Header.css";
-import hpLogo from "../images/hpLogo.png";
+import rmicon from "../images/rmicon.svg";
```

Comment:
I think `rmicon` is to generic for a good name. And there is `icon` in it, but it is used as a logo.